packages/amplify-category-api/src/category-utils/show-auth-acm.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  AccessControlMatrix,
+  AuthRule,
+  ModelOperation,
+  MODEL_OPERATIONS,
+  DEFAULT_GROUPS_FIELD,
+  DEFAULT_GROUP_CLAIM,
+  DEFAULT_OWNER_FIELD,
+  getAuthDirectiveRules,
+} from '@aws-amplify/graphql-auth-transformer';
+import { $TSContext, stateManager, pathManager, FeatureFlags } from '@aws-amplify/amplify-cli-core';
+import { parse, ObjectTypeDefinitionNode, DirectiveNode, FieldDefinitionNode } from 'graphql';
+import { printer } from '@aws-amplify/amplify-prompts';
+import { DirectiveWrapper } from '@aws-amplify/graphql-transformer-core';
+import { readProjectSchema } from 'graphql-transformer-core';
```

Comment:
We might want to refactor this either in this PR or otherwise so that we eliminate dependencies on V1 transformer core.